src/cek/CekMachine.js

```diff
@@ -41,19 +44,31 @@ export class CekMachine {
      */
     state
 
+    /**
+     * @type {UplcLoggingI}
+     */
+    diagnostics
+
+    /**
+     * @type {UplcLoggingI["logPrint"]}
+     */
+    logPrint
```

Comment:
I might've missed something, but I think `this.logPrint` is unused.

Reply:
right